moderation/commands.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 func MBaseCmd(cmdData *dcmd.Data, targetID int64) (config *Config, targetUser *discordgo.User, err error) {
-	config, err = GetConfigOrDefault(cmdData.GuildData.GS.ID)
+	config, err = FetchConfig(cmdData.GuildData.GS.ID)
```

Comment:
Commands run on the bot process and so I think it is safe (and a performance optimization) to call `BotCachedGetConfig` here. I have not made that change in this PR in the interest of keeping its scope to automated, obviously safe changes only.